src/converter-gen/testing/definition-maps/profile-reduce-map-output.xml

```diff
@@ -0,0 +1,7009 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Non-blocking: as always, I will recommend deleting intermediate outputs if possible to make the noise on changes as tests evolve cleaner and not confuse novice devs in the repo, but I defer to you as the maintainer on this project. 😄

Reply:
Hi, @aj-stein-nist . If I'm interpreting this PR correctly, I introduced this file in #87. The purpose of the file is to provide markup that XSpec can use for unit testing of pieces of this pipeline. Can your concern about novice devs be addressed by code comments in the file or a README in the directory?